apps/fiona/fiona.cpp

```diff
@@ -1254,7 +1254,7 @@ getFoundCorrections(
     unsigned numCorrections = 0;
 
     TValue currentPos = firstCorrectionForRead[erroneousReadId];
-    while (currentPos != maxValue<TValue>())
+    while (currentPos != MaxValue<TValue>::VALUE)
```

Comment:
Not every type is a primitive type that can be resolved with numeric_limits. For example When a DNA/DNA5 alphabet is used the MaxValue<> function will return 'T'/'N'. To avoid any errors I only replaced primitive types with numeric_limits and used the seqan meta function for generic types.